app/views/userProfile.scala.html

```diff
@@ -41,6 +45,35 @@
             </div>
         }
 
+        <div class="modal fade" id="createTeamModal" tabindex="-1" role="dialog" aria-labelledby="createTeamModalLabel" aria-hidden="true">
```

Comment:
My default assumption is that these ids and such should be in kebab-case instead of camelCase. I'm assuming that you're filling this in based off of previous code that we had? I'd try to switch to kebab-case and see if there's an issue with that! Feels weird to be mixing the cases here if we don't have to